apps/prairielearn/src/pages/studentAssessmentInstance/studentAssessmentInstance.js

```diff
@@ -291,7 +292,8 @@ router.get(
         }
       }
     }
-    res.render(import.meta.filename.replace(/\.js$/, '.ejs'), res.locals);
+
+    res.send(StudentAssessmentInstance({ resLocals: res.locals }));
```

Comment:
Note that I did not yet move any data out of `res.locals` and into separate properties here. This page still uses enough very complex partials that I don't yet feel comfortable doing that. Once everything downstream of the page is refactored from EJS partials to TypeScript, I'll be able to confidently make this change.